docs/glossary.md

```diff
@@ -290,6 +340,10 @@ A wallet mode in which only blocks related to addresses owned by the wallet are
 
 An interval of 2016 blocks (~1 week) which is used to determine if a vote on concensus rules can begin. Before a vote on consensus rule changes can begin, 75% of tickets that vote during a Stake Version Interval (SVI) must be using software that contains the latent software change being proposed.
 
+#### Stakebase transaction
+
+Every block which is mined will contain a stakebase transaction for each ticket which voted on that block. Stakebase transactions have two inputs, the Decred which was spent to purchase the ticket, and the newly created Decred constituting the reward for voting. Stakebase transactions are also created when an expired or missed ticket is revoked, however there will be no voting reward created in these cases. Stakebase transactions belong to the stake transaction tree, which means that these transactions cannot be rejected by proof-of-stake voters, even if they vote to reject the block which contains them.
```

Comment:
Again, the specific number of inputs and outputs is almost positively going to change over time.  I highly suggest to describe the purpose, not the specific technical details of the transaction structure.